examples/prometheus/index.js

```diff
@@ -3,16 +3,13 @@
 const { MeterProvider } = require('@opentelemetry/sdk-metrics-base');
 const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
 
-const exporter = new PrometheusExporter(
-  {
-    startServer: true,
```

Comment:
I could instead replace with `preventStartServer: false` but it is implicit so removing altogether